subt/tools/validator.py

```diff
@@ -103,63 +103,59 @@ def main():
     parser.add_argument('--draw', help="draw debug results", action='store_true')
     args = parser.parse_args()
 
-    if args.logfile is None:
+    if args.logfiles == []:
```

Comment:
`if not args.logfiles` would have worked for both `None` and `[]`, right?

Reply:
Hmm. The thing is that when we had `nargs='?'` the default values wa `None` but with `nargs='*'` it is `[]`. I tend to be rather explicit so I haven't even realized that `if not args.logfiles` would have worked (I actually had to fire up python REPL and try it to be sure).